src/Raven.Server/Documents/Indexes/Static/StaticIndexBase.cs

```diff
@@ -560,7 +560,13 @@ private static object VectorFromText(IndexField indexField, object value)
             {
                 var vectorList = new List<VectorValue>();
                 foreach (var item in enumerable)
+                {
+                    if (item is null)
```

Comment:
Would it make sense to index this as "null" instead of skipping it?
If yes, maybe we should have some configuration option..

Reply:
Changed